app/src/main/java/com/sunasterisk/fooddaily/ui/main/MainActivity.kt

```diff
@@ -1,24 +1,92 @@
 package com.sunasterisk.fooddaily.ui.main
 
+import android.content.Context
+import android.content.Intent
+import android.os.Parcelable
+import android.view.MenuItem
+import androidx.fragment.app.Fragment
+import com.google.android.material.bottomnavigation.BottomNavigationView
 import com.sunasterisk.fooddaily.R
+import com.sunasterisk.fooddaily.data.model.FoodDetail
 import com.sunasterisk.fooddaily.ui.base.BaseActivity
+import com.sunasterisk.fooddaily.ui.fragment.CollectionFragment
+import com.sunasterisk.fooddaily.ui.fragment.CookingFragment
 import com.sunasterisk.fooddaily.ui.fragment.HomeFragment
+import com.sunasterisk.fooddaily.utils.Constants
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlinx.android.synthetic.main.custom_action_bar.*
 
 class MainActivity : BaseActivity() {
 
     override val layoutRes: Int = R.layout.activity_main
 
+    private val bottomNavSelected = BottomNavigationView.OnNavigationItemSelectedListener { item ->
+        when (item.itemId) {
+            R.id.navigation_home -> {
+                switchScreen(item, HomeFragment())
+                true
+            }
+            R.id.navigation_collection -> {
+                switchScreen(item, CollectionFragment())
+                true
+            }
+            R.id.navigation_cooking -> {
+                switchScreen(item, CookingFragment())
+                true
+            }
+            else -> false
+        }
+    }
+
     override fun initView() {
+        createFoodsAll()
+        createFoodsDaily()
         initFragment()
+        initBottomNavigation()
+    }
+
+    private fun createFoodsAll() {
+        foodsAll = intent.getParcelableArrayListExtra(Constants.EXTRA_FOOD_LIST)
+    }
+
+    private fun createFoodsDaily() {
+        for (i in 5 until foodsAll.size) {
+            if (i % 3 == 0) {
+                foodsDaily.add(foodsAll[i])
+            }
+        }
     }
```

Comment:
hàm này dùng để làm gì đây

Reply:
mình tạo 1 list nhỏ cho RecyclerView DailyMenu

Reply:
giá trị này fix cứng hay sao hay như nào 
list nhỏ về gì 
sao dữ liệu lại xử lý trên View,

Reply:
Màn hình splash của mình request api và nhận về 1 list to các food rồi chuyển qua màn hình main. Màn hình main thì có nhiều fragment, nên mình quyết định để luôn trong main mà không chuyển list đó tới từng fragment nữa. List nhỏ này là mình lấy ra 1 vài food trong list to để đưa vào daily menu, list to thì mình đặt vào trong other list. Mình xử lý như vậy ổn không ?

Reply:
T nghĩ là như thế cũng được, nhưng mà mình không nên để trong compaion object 
Nếu fragment nào cần dữ liệu, mình sẽ truyền nó qua argument của fragment đó 
Giống như cách ông truyền từ mà Splash sang màn Main